styles/Rules.module.css

```diff
@@ -0,0 +1,25 @@
+.container {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.menuItems {
+  margin-top: 4em;
+  max-width: 90vw;
+  display: grid;
+  grid-auto-flow: column;
+  row-gap: 1.5em;
+  align-items: start;
+  justify-items: center;
+  column-gap: 3.5em;
+}
+
+.rules {
+  font-size: 1.5em;
+  max-width: 40rem;
+  overflow-y: auto;
+  max-height: 60vh;
```

Comment:
You should look for a way to remove these properties, maybe even try to put everything into one single grid.